source/shared/cpp/ObjectModel/MarkDownBlockParser.cpp

```diff
@@ -263,7 +263,7 @@ bool EmphasisParser::IsLeftEmphasisDelimiter(const char ch) const
 
 bool EmphasisParser::IsRightEmphasisDelimiter(const char ch) const
 {
-    if ((ch == EOF || MarkDownBlockParser::IsSpace(ch)) && (m_lookBehind != DelimiterType::WhiteSpace) &&
+    if ((ch == EOF || ch == '\xff' || MarkDownBlockParser::IsSpace(ch)) && (m_lookBehind != DelimiterType::WhiteSpace) &&
```

Comment:
I think this is now a complicated enough condition to warrant a quick explanatory code comment pls!